eve/main.yml

```diff
@@ -30,7 +30,23 @@ stages:
         retryFetch: true
         haltOnFailure: true
     - ShellCommand:
-        name: 'Run test-suite'
+        name: 'Init node'
         haltOnFailure: true
         command: |-
-          bash ./vendor/bash_unit/bash_unit tests/single-node/test.sh
+          bash tests/single-node/init.sh
```

Comment:
The current install test comes with its own inventory (which is fine), which declares `/dev/loop0` as storage device, so IMHO it's up to the test to set up this requirement, not some other script ran before the test.

Reply:
But this one need sudo, and to be on run on the VM. Having something external to the VM, would:
- require to copy git repository over ssh and then run test (which is a performance penalty)
- cannot be done right now by our releng team, and CI environment.

I prefer having a simple contract with tox: provide an inventory, it will run test on it; and then rely on eve orchestration to bootstrap platform and run test on it (sure some test, like failover would never run on single-node test, but could be skipped easily with a `pytest.mark.skipif(condition_on_inventory)`

Reply:
OK, then let's add a skip if `/dev/loop0` doesn't exist, rather than make tests fail.

Reply:
`/dev/loop0` is what eve single-node provide, but there is not guarantee for that. I would add some precheck, in the `given` step (something like `playbook/deploy.yml -t assertion`)

Reply:
Ahaaaa, I may be getting it now.

So basically what you say is: the environment is prepared by Eve (OK!), and the test runs in this environment (OK!), and the test relies on some inventory which is suitable for the test (OK).

In that case, the inventory should be stored in `eve/...`